static/GwardiaHub/tailwind_custom_styles.js

```diff
@@ -0,0 +1,17 @@
+// Used for custom styles in templates
+
+tailwind.config = {
+      theme: {
+        extend: {
+          colors: {
+            main_bg: '#0d1017',
+            border_bg: '#484f58',
```

Comment:
this border is quite ugly...

Reply:
You mean its color?

Reply:
like what's the purpose

Reply:
Welp, IMO background with border looks better than without